### ISSUE TITLE
Button needs support for "aria-controls" and "aria-expanded"

### ISSUE THREAD

**Issue by smhigley** [dojo/widgets#108]
**Bug**

The button widget currently supports `hasPopup`, but doesn't have a way to set `aria-controls` and `aria-expanded`, which would often be required for a button triggering a popup.
